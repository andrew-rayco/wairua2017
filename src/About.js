import React from 'react'

const About = () => {
  return (
    <div className="about">
      <h2>About Us</h2>
      <div className="row">
        <div className="four columns">
          <img src="http://placehold.it/400x300" alt="Shane"/>
        </div>
        <div className="eight columns text-left">
          <h3>Shane</h3>
          <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Nulla saepe ratione itaque cum amet, recusandae non perspiciatis quasi, beatae soluta, perferendis tempora minima nesciunt quaerat! Laboriosam animi, optio sit quidem.</p>
        </div>
      </div>
      <div className="row">
        <div className="four columns">
          <img src="http://placehold.it/400x300" alt="Shane"/>
        </div>
        <div className="eight columns text-left">
          <h3>Jen</h3>
          <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Nulla saepe ratione itaque cum amet, recusandae non perspiciatis quasi, beatae soluta, perferendis tempora minima nesciunt quaerat! Laboriosam animi, optio sit quidem.</p>
        </div>
      </div>
    </div>
  )
}

export default About
