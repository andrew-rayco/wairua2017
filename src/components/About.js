import React from 'react'

const About = () => {
  return (
    <section className="about top-margin">

      <h3 className="title">About Us</h3>
        <p>Wairua Adventures was born out of a love and passion for helping others reconnect with their adventurous spirit. We have created unique and exciting adventure based retreats that combine our love of exploring  the outdoors with holistic practices to balance your mind, body and spirit.</p>
        <p>We have aligned with some incredibly talented and passionate people who will share their wisdom and knowledge in a variety of creative workshops. Covering valuable topics such as Mindfulness, Meditation &amp; Spirituality, Nutrition and Healthy Lifestyle Choices, Motivational Speaking and Life Coaching, Traditional Chinese Medicine, Yoga, Martial Arts, Qi Gong, Tai Chi and more!</p>
        <p>At Wairua Adventures all of our retreats have been lovingly crafted to offer you the perfect blend of adventure and to reawaken your spirit. So take some time out from the ‘machine world’ and have lots of fun being fully present and in the moment with a group of like minded, adventure seeking souls!</p>

        <div className="bios">
          <div className="about-image">
            <img src="../images/shane.jpg" alt="Shane"/>
          </div>
          <div className="about-text">
            <h4>Shane</h4>
            <p>Laboriosam animi, optio sit quidem. Laboriosam lorem ipsum dolor sit amet, consectetur adipisicing elit. Nulla saepe ratione itaque cum amet, recusandae non perspiciatis quasi, beatae soluta, perferendis tempora minima nesciunt quaerat! Laboriosam animi, optio sit quidem.</p>
          </div>


          <div className="about-image">
            <img src="../images/jen.jpg" alt="Jen"/>
          </div>
          <div className="about-text">
            <h4>Jen</h4>
            <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Nulla saepe ratione itaque cum amet, recusandae non perspiciatis quasi, beatae soluta, perferendis tempora minima nesciunt quaerat! Laboriosam animi, optio sit quidem.</p>
          </div>
        </div>

    </section>
  )
}

export default About
