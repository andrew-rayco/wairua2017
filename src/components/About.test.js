import React from 'react'
import { shallow } from 'enzyme'
import About from './About'

describe('<About /> component', () => {
  const app = shallow(<About />)

  it('renders correctly', () => {
    expect(app).toMatchSnapshot()
  })
})
