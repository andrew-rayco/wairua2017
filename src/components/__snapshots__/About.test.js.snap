// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<About /> component renders correctly 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <About />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": null,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <h3
          className="title"
>
          About Us
</h3>,
        <p>
          Wairua Adventures was born out of a love and passion for helping others reconnect with their adventurous spirit. We have created unique and exciting adventure based retreats that combine our love of exploring  the outdoors with holistic practices to balance your mind, body and spirit.
</p>,
        <p>
          We have aligned with some incredibly talented and passionate people who will share their wisdom and knowledge in a variety of creative workshops. Covering valuable topics such as Mindfulness, Meditation & Spirituality, Nutrition and Healthy Lifestyle Choices, Motivational Speaking and Life Coaching, Traditional Chinese Medicine, Yoga, Martial Arts, Qi Gong, Tai Chi and more!
</p>,
        <p>
          At Wairua Adventures all of our retreats have been lovingly crafted to offer you the perfect blend of adventure and to reawaken your spirit. So take some time out from the ‘machine world’ and have lots of fun being fully present and in the moment with a group of like minded, adventure seeking souls!
</p>,
        <div
          className="bios"
>
          <div
                    className="about-image"
          >
                    <img
                              alt="Shane"
                              src="../images/shane.jpg"
                    />
          </div>
          <div
                    className="about-text"
          >
                    <h4>
                              Shane
                    </h4>
                    <p>
                              Laboriosam animi, optio sit quidem. Laboriosam lorem ipsum dolor sit amet, consectetur adipisicing elit. Nulla saepe ratione itaque cum amet, recusandae non perspiciatis quasi, beatae soluta, perferendis tempora minima nesciunt quaerat! Laboriosam animi, optio sit quidem.
                    </p>
          </div>
          <div
                    className="about-image"
          >
                    <img
                              alt="Jen"
                              src="../images/jen.jpg"
                    />
          </div>
          <div
                    className="about-text"
          >
                    <h4>
                              Jen
                    </h4>
                    <p>
                              Lorem ipsum dolor sit amet, consectetur adipisicing elit. Nulla saepe ratione itaque cum amet, recusandae non perspiciatis quasi, beatae soluta, perferendis tempora minima nesciunt quaerat! Laboriosam animi, optio sit quidem.
                    </p>
          </div>
</div>,
      ],
      "className": "about top-margin",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": null,
        "nodeType": "host",
        "props": Object {
          "children": "About Us",
          "className": "title",
        },
        "ref": null,
        "rendered": "About Us",
        "type": "h3",
      },
      Object {
        "instance": null,
        "key": null,
        "nodeType": "host",
        "props": Object {
          "children": "Wairua Adventures was born out of a love and passion for helping others reconnect with their adventurous spirit. We have created unique and exciting adventure based retreats that combine our love of exploring  the outdoors with holistic practices to balance your mind, body and spirit.",
        },
        "ref": null,
        "rendered": "Wairua Adventures was born out of a love and passion for helping others reconnect with their adventurous spirit. We have created unique and exciting adventure based retreats that combine our love of exploring  the outdoors with holistic practices to balance your mind, body and spirit.",
        "type": "p",
      },
      Object {
        "instance": null,
        "key": null,
        "nodeType": "host",
        "props": Object {
          "children": "We have aligned with some incredibly talented and passionate people who will share their wisdom and knowledge in a variety of creative workshops. Covering valuable topics such as Mindfulness, Meditation & Spirituality, Nutrition and Healthy Lifestyle Choices, Motivational Speaking and Life Coaching, Traditional Chinese Medicine, Yoga, Martial Arts, Qi Gong, Tai Chi and more!",
        },
        "ref": null,
        "rendered": "We have aligned with some incredibly talented and passionate people who will share their wisdom and knowledge in a variety of creative workshops. Covering valuable topics such as Mindfulness, Meditation & Spirituality, Nutrition and Healthy Lifestyle Choices, Motivational Speaking and Life Coaching, Traditional Chinese Medicine, Yoga, Martial Arts, Qi Gong, Tai Chi and more!",
        "type": "p",
      },
      Object {
        "instance": null,
        "key": null,
        "nodeType": "host",
        "props": Object {
          "children": "At Wairua Adventures all of our retreats have been lovingly crafted to offer you the perfect blend of adventure and to reawaken your spirit. So take some time out from the ‘machine world’ and have lots of fun being fully present and in the moment with a group of like minded, adventure seeking souls!",
        },
        "ref": null,
        "rendered": "At Wairua Adventures all of our retreats have been lovingly crafted to offer you the perfect blend of adventure and to reawaken your spirit. So take some time out from the ‘machine world’ and have lots of fun being fully present and in the moment with a group of like minded, adventure seeking souls!",
        "type": "p",
      },
      Object {
        "instance": null,
        "key": null,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <div
              className="about-image"
>
              <img
                            alt="Shane"
                            src="../images/shane.jpg"
              />
</div>,
            <div
              className="about-text"
>
              <h4>
                            Shane
              </h4>
              <p>
                            Laboriosam animi, optio sit quidem. Laboriosam lorem ipsum dolor sit amet, consectetur adipisicing elit. Nulla saepe ratione itaque cum amet, recusandae non perspiciatis quasi, beatae soluta, perferendis tempora minima nesciunt quaerat! Laboriosam animi, optio sit quidem.
              </p>
</div>,
            <div
              className="about-image"
>
              <img
                            alt="Jen"
                            src="../images/jen.jpg"
              />
</div>,
            <div
              className="about-text"
>
              <h4>
                            Jen
              </h4>
              <p>
                            Lorem ipsum dolor sit amet, consectetur adipisicing elit. Nulla saepe ratione itaque cum amet, recusandae non perspiciatis quasi, beatae soluta, perferendis tempora minima nesciunt quaerat! Laboriosam animi, optio sit quidem.
              </p>
</div>,
          ],
          "className": "bios",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": null,
            "nodeType": "host",
            "props": Object {
              "children": <img
                alt="Shane"
                src="../images/shane.jpg"
/>,
              "className": "about-image",
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": null,
              "nodeType": "host",
              "props": Object {
                "alt": "Shane",
                "src": "../images/shane.jpg",
              },
              "ref": null,
              "rendered": null,
              "type": "img",
            },
            "type": "div",
          },
          Object {
            "instance": null,
            "key": null,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <h4>
                  Shane
</h4>,
                <p>
                  Laboriosam animi, optio sit quidem. Laboriosam lorem ipsum dolor sit amet, consectetur adipisicing elit. Nulla saepe ratione itaque cum amet, recusandae non perspiciatis quasi, beatae soluta, perferendis tempora minima nesciunt quaerat! Laboriosam animi, optio sit quidem.
</p>,
              ],
              "className": "about-text",
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": null,
                "nodeType": "host",
                "props": Object {
                  "children": "Shane",
                },
                "ref": null,
                "rendered": "Shane",
                "type": "h4",
              },
              Object {
                "instance": null,
                "key": null,
                "nodeType": "host",
                "props": Object {
                  "children": "Laboriosam animi, optio sit quidem. Laboriosam lorem ipsum dolor sit amet, consectetur adipisicing elit. Nulla saepe ratione itaque cum amet, recusandae non perspiciatis quasi, beatae soluta, perferendis tempora minima nesciunt quaerat! Laboriosam animi, optio sit quidem.",
                },
                "ref": null,
                "rendered": "Laboriosam animi, optio sit quidem. Laboriosam lorem ipsum dolor sit amet, consectetur adipisicing elit. Nulla saepe ratione itaque cum amet, recusandae non perspiciatis quasi, beatae soluta, perferendis tempora minima nesciunt quaerat! Laboriosam animi, optio sit quidem.",
                "type": "p",
              },
            ],
            "type": "div",
          },
          Object {
            "instance": null,
            "key": null,
            "nodeType": "host",
            "props": Object {
              "children": <img
                alt="Jen"
                src="../images/jen.jpg"
/>,
              "className": "about-image",
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": null,
              "nodeType": "host",
              "props": Object {
                "alt": "Jen",
                "src": "../images/jen.jpg",
              },
              "ref": null,
              "rendered": null,
              "type": "img",
            },
            "type": "div",
          },
          Object {
            "instance": null,
            "key": null,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <h4>
                  Jen
</h4>,
                <p>
                  Lorem ipsum dolor sit amet, consectetur adipisicing elit. Nulla saepe ratione itaque cum amet, recusandae non perspiciatis quasi, beatae soluta, perferendis tempora minima nesciunt quaerat! Laboriosam animi, optio sit quidem.
</p>,
              ],
              "className": "about-text",
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": null,
                "nodeType": "host",
                "props": Object {
                  "children": "Jen",
                },
                "ref": null,
                "rendered": "Jen",
                "type": "h4",
              },
              Object {
                "instance": null,
                "key": null,
                "nodeType": "host",
                "props": Object {
                  "children": "Lorem ipsum dolor sit amet, consectetur adipisicing elit. Nulla saepe ratione itaque cum amet, recusandae non perspiciatis quasi, beatae soluta, perferendis tempora minima nesciunt quaerat! Laboriosam animi, optio sit quidem.",
                },
                "ref": null,
                "rendered": "Lorem ipsum dolor sit amet, consectetur adipisicing elit. Nulla saepe ratione itaque cum amet, recusandae non perspiciatis quasi, beatae soluta, perferendis tempora minima nesciunt quaerat! Laboriosam animi, optio sit quidem.",
                "type": "p",
              },
            ],
            "type": "div",
          },
        ],
        "type": "div",
      },
    ],
    "type": "section",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": null,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <h3
            className="title"
>
            About Us
</h3>,
          <p>
            Wairua Adventures was born out of a love and passion for helping others reconnect with their adventurous spirit. We have created unique and exciting adventure based retreats that combine our love of exploring  the outdoors with holistic practices to balance your mind, body and spirit.
</p>,
          <p>
            We have aligned with some incredibly talented and passionate people who will share their wisdom and knowledge in a variety of creative workshops. Covering valuable topics such as Mindfulness, Meditation & Spirituality, Nutrition and Healthy Lifestyle Choices, Motivational Speaking and Life Coaching, Traditional Chinese Medicine, Yoga, Martial Arts, Qi Gong, Tai Chi and more!
</p>,
          <p>
            At Wairua Adventures all of our retreats have been lovingly crafted to offer you the perfect blend of adventure and to reawaken your spirit. So take some time out from the ‘machine world’ and have lots of fun being fully present and in the moment with a group of like minded, adventure seeking souls!
</p>,
          <div
            className="bios"
>
            <div
                        className="about-image"
            >
                        <img
                                    alt="Shane"
                                    src="../images/shane.jpg"
                        />
            </div>
            <div
                        className="about-text"
            >
                        <h4>
                                    Shane
                        </h4>
                        <p>
                                    Laboriosam animi, optio sit quidem. Laboriosam lorem ipsum dolor sit amet, consectetur adipisicing elit. Nulla saepe ratione itaque cum amet, recusandae non perspiciatis quasi, beatae soluta, perferendis tempora minima nesciunt quaerat! Laboriosam animi, optio sit quidem.
                        </p>
            </div>
            <div
                        className="about-image"
            >
                        <img
                                    alt="Jen"
                                    src="../images/jen.jpg"
                        />
            </div>
            <div
                        className="about-text"
            >
                        <h4>
                                    Jen
                        </h4>
                        <p>
                                    Lorem ipsum dolor sit amet, consectetur adipisicing elit. Nulla saepe ratione itaque cum amet, recusandae non perspiciatis quasi, beatae soluta, perferendis tempora minima nesciunt quaerat! Laboriosam animi, optio sit quidem.
                        </p>
            </div>
</div>,
        ],
        "className": "about top-margin",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": null,
          "nodeType": "host",
          "props": Object {
            "children": "About Us",
            "className": "title",
          },
          "ref": null,
          "rendered": "About Us",
          "type": "h3",
        },
        Object {
          "instance": null,
          "key": null,
          "nodeType": "host",
          "props": Object {
            "children": "Wairua Adventures was born out of a love and passion for helping others reconnect with their adventurous spirit. We have created unique and exciting adventure based retreats that combine our love of exploring  the outdoors with holistic practices to balance your mind, body and spirit.",
          },
          "ref": null,
          "rendered": "Wairua Adventures was born out of a love and passion for helping others reconnect with their adventurous spirit. We have created unique and exciting adventure based retreats that combine our love of exploring  the outdoors with holistic practices to balance your mind, body and spirit.",
          "type": "p",
        },
        Object {
          "instance": null,
          "key": null,
          "nodeType": "host",
          "props": Object {
            "children": "We have aligned with some incredibly talented and passionate people who will share their wisdom and knowledge in a variety of creative workshops. Covering valuable topics such as Mindfulness, Meditation & Spirituality, Nutrition and Healthy Lifestyle Choices, Motivational Speaking and Life Coaching, Traditional Chinese Medicine, Yoga, Martial Arts, Qi Gong, Tai Chi and more!",
          },
          "ref": null,
          "rendered": "We have aligned with some incredibly talented and passionate people who will share their wisdom and knowledge in a variety of creative workshops. Covering valuable topics such as Mindfulness, Meditation & Spirituality, Nutrition and Healthy Lifestyle Choices, Motivational Speaking and Life Coaching, Traditional Chinese Medicine, Yoga, Martial Arts, Qi Gong, Tai Chi and more!",
          "type": "p",
        },
        Object {
          "instance": null,
          "key": null,
          "nodeType": "host",
          "props": Object {
            "children": "At Wairua Adventures all of our retreats have been lovingly crafted to offer you the perfect blend of adventure and to reawaken your spirit. So take some time out from the ‘machine world’ and have lots of fun being fully present and in the moment with a group of like minded, adventure seeking souls!",
          },
          "ref": null,
          "rendered": "At Wairua Adventures all of our retreats have been lovingly crafted to offer you the perfect blend of adventure and to reawaken your spirit. So take some time out from the ‘machine world’ and have lots of fun being fully present and in the moment with a group of like minded, adventure seeking souls!",
          "type": "p",
        },
        Object {
          "instance": null,
          "key": null,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <div
                className="about-image"
>
                <img
                                alt="Shane"
                                src="../images/shane.jpg"
                />
</div>,
              <div
                className="about-text"
>
                <h4>
                                Shane
                </h4>
                <p>
                                Laboriosam animi, optio sit quidem. Laboriosam lorem ipsum dolor sit amet, consectetur adipisicing elit. Nulla saepe ratione itaque cum amet, recusandae non perspiciatis quasi, beatae soluta, perferendis tempora minima nesciunt quaerat! Laboriosam animi, optio sit quidem.
                </p>
</div>,
              <div
                className="about-image"
>
                <img
                                alt="Jen"
                                src="../images/jen.jpg"
                />
</div>,
              <div
                className="about-text"
>
                <h4>
                                Jen
                </h4>
                <p>
                                Lorem ipsum dolor sit amet, consectetur adipisicing elit. Nulla saepe ratione itaque cum amet, recusandae non perspiciatis quasi, beatae soluta, perferendis tempora minima nesciunt quaerat! Laboriosam animi, optio sit quidem.
                </p>
</div>,
            ],
            "className": "bios",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": null,
              "nodeType": "host",
              "props": Object {
                "children": <img
                  alt="Shane"
                  src="../images/shane.jpg"
/>,
                "className": "about-image",
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": null,
                "nodeType": "host",
                "props": Object {
                  "alt": "Shane",
                  "src": "../images/shane.jpg",
                },
                "ref": null,
                "rendered": null,
                "type": "img",
              },
              "type": "div",
            },
            Object {
              "instance": null,
              "key": null,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  <h4>
                    Shane
</h4>,
                  <p>
                    Laboriosam animi, optio sit quidem. Laboriosam lorem ipsum dolor sit amet, consectetur adipisicing elit. Nulla saepe ratione itaque cum amet, recusandae non perspiciatis quasi, beatae soluta, perferendis tempora minima nesciunt quaerat! Laboriosam animi, optio sit quidem.
</p>,
                ],
                "className": "about-text",
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": null,
                  "nodeType": "host",
                  "props": Object {
                    "children": "Shane",
                  },
                  "ref": null,
                  "rendered": "Shane",
                  "type": "h4",
                },
                Object {
                  "instance": null,
                  "key": null,
                  "nodeType": "host",
                  "props": Object {
                    "children": "Laboriosam animi, optio sit quidem. Laboriosam lorem ipsum dolor sit amet, consectetur adipisicing elit. Nulla saepe ratione itaque cum amet, recusandae non perspiciatis quasi, beatae soluta, perferendis tempora minima nesciunt quaerat! Laboriosam animi, optio sit quidem.",
                  },
                  "ref": null,
                  "rendered": "Laboriosam animi, optio sit quidem. Laboriosam lorem ipsum dolor sit amet, consectetur adipisicing elit. Nulla saepe ratione itaque cum amet, recusandae non perspiciatis quasi, beatae soluta, perferendis tempora minima nesciunt quaerat! Laboriosam animi, optio sit quidem.",
                  "type": "p",
                },
              ],
              "type": "div",
            },
            Object {
              "instance": null,
              "key": null,
              "nodeType": "host",
              "props": Object {
                "children": <img
                  alt="Jen"
                  src="../images/jen.jpg"
/>,
                "className": "about-image",
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": null,
                "nodeType": "host",
                "props": Object {
                  "alt": "Jen",
                  "src": "../images/jen.jpg",
                },
                "ref": null,
                "rendered": null,
                "type": "img",
              },
              "type": "div",
            },
            Object {
              "instance": null,
              "key": null,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  <h4>
                    Jen
</h4>,
                  <p>
                    Lorem ipsum dolor sit amet, consectetur adipisicing elit. Nulla saepe ratione itaque cum amet, recusandae non perspiciatis quasi, beatae soluta, perferendis tempora minima nesciunt quaerat! Laboriosam animi, optio sit quidem.
</p>,
                ],
                "className": "about-text",
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": null,
                  "nodeType": "host",
                  "props": Object {
                    "children": "Jen",
                  },
                  "ref": null,
                  "rendered": "Jen",
                  "type": "h4",
                },
                Object {
                  "instance": null,
                  "key": null,
                  "nodeType": "host",
                  "props": Object {
                    "children": "Lorem ipsum dolor sit amet, consectetur adipisicing elit. Nulla saepe ratione itaque cum amet, recusandae non perspiciatis quasi, beatae soluta, perferendis tempora minima nesciunt quaerat! Laboriosam animi, optio sit quidem.",
                  },
                  "ref": null,
                  "rendered": "Lorem ipsum dolor sit amet, consectetur adipisicing elit. Nulla saepe ratione itaque cum amet, recusandae non perspiciatis quasi, beatae soluta, perferendis tempora minima nesciunt quaerat! Laboriosam animi, optio sit quidem.",
                  "type": "p",
                },
              ],
              "type": "div",
            },
          ],
          "type": "div",
        },
      ],
      "type": "section",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactFifteenAdapter {
      "options": Object {
        "supportPrevContextArgumentOfComponentDidUpdate": true,
      },
    },
  },
}
`;
